decidim-budgets/app/views/decidim/budgets/projects/_order.html.erb

```diff
@@ -1 +1 @@
-<%= order_selector available_orders, i18n_scope: "decidim.budgets.projects.orders" %>
+<%= order_selector available_orders, i18n_scope: "decidim.budgets.projects.orders", css_class: "items-start pl-[18px]" %>
```

Comment:
feedback: avoid pl-[18px] as it will break tailwind variables. I suggest to find something `pl-4`.
If you want to use [18px], please be sure it is used in other part of decidim (and shouldn't)